src/GitHub_Updater/Base.php

```diff
@@ -585,13 +594,35 @@ public function update_row_enclosure( $repo_name, $type, $branch_switcher = fals
 	public function make_branch_switch_row( $data, $config ) {
 		$rollback = empty( $config[ $data['slug'] ]->rollback ) ? [] : $config[ $data['slug'] ]->rollback;
 
+		// Make the branch switch row visually appear as if it is contained with the plugin/theme's row.
+		// We have to use JS for this because of the way:
+		// 1) the @class of the list table row is not filterabled; and
+		// 2) the list table CSS is written.
+		$data_attr = $config[ $data['slug'] ]->type ? 'data-plugin' : 'data-slug';
+		$file      = $config[ $data['slug'] ]->type ? $config[ $data['slug'] ]->file : $config[ $data['slug'] ]->slug;
+		echo '<script>';
+		// Remove the bottom "line" for the plugin's row.
+		printf(
+			"jQuery( 'tr:not([id])[{$data_attr}=\"%s\"]' ).addClass( 'update' );",
+			$file
+		);
+		// Removes the bottom "line" for the shinny update row (if any).
```

Comment:
typo, should be 'shiny'

Reply:
fixed in https://github.com/afragen/github-updater/pull/898/commits/c7a18168b879b567adfd3e78aace1a14cfc2e160